### DISCUSSION TITLE
How to return model data as an embed?

### DISCUSSION THREAD

**Discussion by zephiowl** [serenity-rs/serenity#2775] [Q&A]
Heya so I've been making a very small lightweight bot to learn serenity and i have something simple going but i want to possible find a way to instead of returning the data as a message, do it as an embed. [heres my project so far](https://github.com/Zephira58/purplewood/blob/main/src/commands/recruit.rs) if anyone has a small code snipped, or can provide some insight as to what exactly i can do that'd be great, as i haven't been able to get it working to great on my end while trying to test locally ♥

PS: Sorry if its a dumb question, i'm far from the best programmer as i'm sure you'll see if you look at my code :p

**Comment by jamesbt365** [comment]
Shouldn't be too much different to what you have already, you already grabbed what data you want to display so you just need to call the `embed` method instead of content, then pass the [Create Embed](https://docs.rs/serenity/0.12.1/serenity/builder/struct.CreateEmbed.html) builder. We have an example on how to use the builders [here](https://github.com/serenity-rs/serenity/blob/f9f3f2c084c94549ff69124ca13731e3c3467b15/examples/e09_create_message_builder/src/main.rs#L20-L33). You only need to call the internal methods for the parts of the embed that you want to add.
